server/controllers/authController.js

```diff
@@ -72,7 +72,7 @@ module.exports = {
         } else {
             let salt = bcrypt.genSaltSync(10);
             let hash = bcrypt.hashSync(password, salt);
-        }
+
 // did they type in the password input ? yes, then hash it regardless : if no input, then revert to old password
 
         console.log(hash)
```

Comment:
You've got a console log of the Hash here, we should probably delete that so the hash isn't logged anywhere.